### ISSUE TITLE
Calcul de la caisse optimal: Wrapper type réordonner les propositions:

### ISSUE THREAD

**Issue by pbuffet** [treeDiM/StackBuilder#99]
![image](https://user-images.githubusercontent.com/5374935/28772737-63614546-75e7-11e7-9109-29be7c66d619.png)

re-ordonner de la façon suivante:
Cardboard, 
Tray,
Paper 
Polyethylene

Est il possible de changer les valeurs par défaut des paramètres Number of wall, thickness, mass/m² en fonction du choix de wrapper?

**Comment by fgasnier71** [comment]
@pbuffet  : Oui! On peut définir des valeurs par défaut en fonction du type de Wrapper.
Peux-tu fournir une valeur par défaut pour épaisseur et masse surfacique pour chacun des wrappers?
